test/Data/Graph/WrapperSpec.hs

```diff
@@ -21,3 +21,29 @@ spec = do
       it "returns the empty list" $ do
         reachableVertices (fromList [] :: Graph Integer ()) 0
           `shouldBe` []
+
+  describe "Eq Graph" $ do
+    context "when the graphs are identical" $ do
+      it "returns True" $ do
+        let g1 = fromList [('a','a',['a','b','c']), ('b','b',['b','c']), ('c','c',['a'])]
```

Comment:
i think the whitespace here is a inconsistent. Sometimes a comma is followed by a space, sometimes it's not. Could you make sure every comma is followed by a space, please?

Reply:
added.